modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -660,6 +662,27 @@ protected Assignment getAssignment(User.ID userID, Application.Name applicationN
     }
 
     /**
+     * If the experiment is a rapid experiment, it checks whether the user cap has already been achieved.
+     * If yes, then it sets the experiment state to paused, refreshes the experiment metadata cache
+     * and returns false i.e. assignment is not allowed in that case.
+     */
+    private boolean checkUserCapForRapidExperiment(Experiment experiment) {
+        if (experiment.getIsRapidExperiment() != null && experiment.getIsRapidExperiment()) {
```

Comment:
@bargenilesh Can we just agree on using the vanilla null check everywhere. As we had discussed in earlier code reviews as well, the Object.isNull() method was introduced explicitly for the streams API in Java 8.